src/displayData.js

```diff
@@ -4,7 +4,7 @@ const displayData = (data) => {
   const scoreData = data.result;
   scoreData.forEach((data) => {
     const listItem = document.createElement('li');
-    listItem.innerText = `${data.user} : ${data.score}`;
+    listItem.innerHTML = `${data.user}<span>${data.score}</span>`;
```

Comment:
- [ ] Please notice that when I attempt to add a name and score It doesn't add the score to the scoreboard. when clicking refresh the names stay the same. please fix this issue and ensure it works as previously.